nre/bpf/nodefw/nodefw-ebpf/rust-toolchain.toml

```diff
@@ -0,0 +1,13 @@
+[toolchain]
+channel = "stable"
+# The source code of rustc, provided by the rust-src component, is needed for
+# building eBPF programs.
+components = [
+    "cargo",
+    "clippy",
+    "rust-docs",
+    "rust-src",
+    "rust-std",
+    "rustc",
+    "rustfmt",
+]
```

Comment:
Can we remove this file completely and instead rely on the repo level one? If we need to add components we can do that. 

also for future reference just having "stable" isn't great because you don't control when the toolchain moves from vX to vX+1